src/loginPage/loginPage.js

```diff
@@ -8,7 +8,7 @@ class LoginPage extends Component {
     constructor(props) {
         super(props);
 
-        this.socket = io('http://localhost:9876');
+        this.socket = io(process.env.LOCAL_HOST_FRONTEND);
```

Comment:
Please default the url and port to localhost:9876 or throw an error if we do not have this config item passed